server/pyproject.toml

```diff
@@ -22,9 +22,11 @@ typing_extensions = "^4.2"
 quart = {version = "^0.17", optional = true}
 hypercorn = {version = "^0.13", optional = true}
 pymongo = {version = ">=3.12.0", optional = true, extras = ["srv", "tls"]}
-snowflake-connector-python = {version = "^2.7.6", extras = ["pandas"], optional = true}
+snowflake-connector-python = {version = "2.7.4", extras = ["pandas"], optional = true}
 geopandas = "^0.10.2"
 PyPika = "^0.48.9"
+redshift-connector = "^2.0.907"
+sqlalchemy-redshift = "^0.8.9"
```

Comment:
Shouldn't these be test requirements ? Once the lib user has translated the query, it's up to him to decide how he executes it

Reply:
correct